src/ast.rs

```diff
@@ -154,6 +154,10 @@ impl Policy {
 pub enum Expression {
     Decl(Declaration),
     Stmt(Statement),
+    // Needed for parse grammar.  If this is ever set, we should have bailed
```

Comment:
"for parsed" or "to parse"?

Reply:
Intention was "parse grammer" as a noun phrase.  I'll update to "parser grammar" which may be more accurate.